core-library/src/main/java/org/silverpeas/core/socialnetwork/invitation/InvitationService.java

```diff
@@ -52,6 +55,8 @@ public static InvitationService get() {
   private InvitationDao invitationDao;
   @Inject
   private RelationShipDao relationShipDao;
+  @Inject
+  private RelationShipEventNotifier relationShipEventNotifier;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)